### ISSUE TITLE
Breaking changes to :id initarg

### ISSUE THREAD

**Issue by notmgsk** [mbrezu/cl-messagepack#20]
Commit https://github.com/mbrezu/cl-messagepack/commit/b305346a719eb766ce71da4603332ca4b9f8220d changed the initarg for defined extension types from `:id` to `messagepack-sym:id` and is thus not backwards compatible. This caused a brief headache that could've been avoided by either avoiding breaking changes by using an equivalent alternative, or introducing semantic versioning.

**Comment by phmarek** [comment]
Yeah, you're right - this change could've been communicated better. \
My thought was to break the (possible) `:ID` duplication fast; as there was no version number before I didn't get the idea to change it ;/

OTOH... I'm not sure how much good the version number would have done; as long as a simple `(QL:UPDATE-DIST)` just installs a newer version only the breakage would've been noticed, I guess. \
People using the `:ID` initarg to configure `CL-MESSAGEPACK` should've noticed immediately - and people using it for their own classes didn't have the conflicting name any more, which was the goal.
